### PR TITLE
Implement string.rjust() ljust() and center().

### PR DESCRIPTION
**Pull request by corona10**
* Support center(), ljust() and rjust().
* TypeError messages are following CPython 2.7.12
* Add unit tests.

### CODE REVIEW

**Comment by corona10** [comment]
@trotterdylan PTAL

File: runtime/str.go [hunk 1]
```diff
@@ -699,6 +741,26 @@ func strRIndex(f *Frame, args Args, _ KWArgs) (*Object, *BaseException) {
...
+
+	if numChars := len(fill); numChars != 1 {
+		return nil, f.RaiseType(TypeErrorType, fmt.Sprintf("rjust() argument 2 must be char, not str"))
+	}
+
+	if len(s) >= width {
+		return NewStr(s).ToObject(), nil
+	}
+
+	return NewStr(pad(s, width-len(s), 0, fill)).ToObject(), nil
```
Comment (trotterdylan) [inline runtime/str.go]: Seems like everything above this line is common to all the methods except for the name. Maybe factor it out into strJustDecodeArgs or something

File: runtime/str_test.go [hunk 2]
```diff
@@ -306,6 +306,11 @@ func TestStrMethods(t *testing.T) {
 		{"capitalize", wrapArgs("вол"), NewStr("вол").ToObject(), nil},
 		{"capitalize", wrapArgs("foobar", 123), nil, mustCreateException(TypeErrorType, "'capitalize' of 'str' requires 1 arguments")},
 		{"capitalize", wrapArgs("ВОЛ"), NewStr("ВОЛ").ToObject(), nil},
+		{"center", wrapArgs("foobar", 10, "#"), NewStr("##foobar##").ToObject(), nil},
```
Comment (trotterdylan) [inline runtime/str_test.go]: How about a test case where the number of characters to pad are odd. Which side should the extra character go on?

**Comment by corona10** [comment]
@trotterdylan Thanks for the review. Updated!

**Pull request merged**